aten/src/ATen/quantized/Quantizer.h

```diff
@@ -155,6 +155,62 @@ struct CAFFE2_API PerChannelAffineQuantizer : public AffineQuantizer {
   const int64_t axis_;
 };
 
+/**
+ * PerRowFloatQParamsQuantizer is the same as PerChannelAffineQuantizer
```

Comment:
can the naming reflect this similarity? e.g. PerChannelAffineFloatZeroPoint

Reply:
Sure, I can call it PerChannelFloatQParamsQuantizer. Will update names in other places too.

Reply:
please make it `PerChannelAffineFloatQParamsQuantizer`, we want to make it clear this is a variant of `PerChannelAffineQuantizer`